modules/building-memcached-operator-using-osdk.adoc

```diff
@@ -165,7 +165,7 @@ The example controller executes the following reconciliation logic for each
 --
 * Create a Memcached Deployment if it does not exist.
 * Ensure that the Deployment size is the same as specified by the `Memcached` CR spec.
-* Update the `Memcached` CR status with the names of the Memcached Pods.
+* Update the `Memcached` CR status with the names of the Memcached pods.
```

Comment:
I'm honestly not sure. That's a bit outside of the scope of this PR since I'm just worrying about these Pod->pod updates for now. Whoever reviews this book when we divide them up will hopefully take a closer look.